src/Umbraco.Web.UI.Client/src/views/propertyeditors/datepicker/datepicker.controller.js

```diff
@@ -24,7 +24,10 @@ function dateTimePickerController($scope, angularHelper, dateHelper, validationM
         };
 
         // map the user config
-        $scope.model.config = angular.extend(config, $scope.model.config);
+        Utilities.extend(config, $scope.model.config);
+
+        // map back to the model
+        $scope.model.config = config;
```

Comment:
The `Utilities.extend` function also returns the merged object, so this can be simplified as:
```suggestion
        $scope.model.config = Utilities.extend(config, $scope.model.config);
```